.circleci/config.yml

```diff
@@ -564,16 +553,3 @@ workflows:
           <<: *on-version-tags
           requires:
             - deploy-release
-
-  nightly:
-    triggers:
-      - schedule:
-          cron: "20 17 * * *"
-          filters:
-            branches:
-              only: master
-    jobs:
-#      - build-macos:
-#          <<: *never # temporarily disabled
```

Comment:
Maybe keep a `dummy` + `<<: *never` just in case we want to do something nightly in the future? (it's impossible to have 0 jobs in a workflow, so dummy is needed if we want to keep the nightly clause).

Reply:
Let's avoid "polluting" our files. If it is redundant, let's remove it.

Reply:
@DvirDukhan Agree, but note that we're on a move towards an inter-module CircleCI form, so diverging from the common structure is not for the common good. But it's ok, I'll approve it.